.github/workflows/int_test.yml

```diff
@@ -68,30 +78,17 @@ permissions:
   id-token: 'write'
 
 env:
-  KV_NAME: "opskv${{ github.run_number }}x"
   RESOURCE_GROUP: "${{ inputs.resource-group }}"
 
 jobs:
-  create_kv:
-    runs-on: ubuntu-22.04
-    steps:
-    - name: "Az CLI login"
-      uses: azure/login@v2
-      with:
-        client-id: ${{ secrets.AZURE_CLIENT_ID }}
-        tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-        subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-    - name: "Create Key Vault for clusters"
-      run: az keyvault create -n ${{ env.KV_NAME }} -g ${{ env.RESOURCE_GROUP }} --enable-rbac-authorization false --tags run_number=${{ github.run_number }}
-
   test:
-    needs: [create_kv]
     outputs:
       CLUSTER_PREFIX: "iotopstest-${{ github.run_number }}"
+      INSTANCE_PREFIX: "iotinsttest-${{ github.run_number }}"
       RESOURCE_GROUP: ${{ env.RESOURCE_GROUP }}
```

Comment:
I don't think these job-level outputs are used anymore, since the cleanup is done inside this job.